camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/partitioner/VintedPartitioner.java

```diff
@@ -23,11 +23,17 @@ public String encodePartition(JobContext context, IEtlKey key) {
         long outfilePartitionMs = EtlMultiOutputFormat.getEtlOutputFileTimePartitionMins(context) * 60000L;
         String portal = key.getPartitionMap().get(PORTAL).toString();
         // get output date formatter for specific portal
-        String portalTimeZone = context.getConfiguration().get(ETL_PORTAL_TIMEZONE + "." + portal, null);
+        String portalTimeZone = context.getConfiguration().get(ETL_PORTAL_TIMEZONE + "." + portal);
+
         if (portalTimeZone == null) {
-            portalTimeZone = context.getConfiguration().get(EtlMultiOutputFormat.ETL_DEFAULT_TIMEZONE, "America/Los_Angeles");
+            throw new RuntimeException("Missing timezone configuration for portal=" + portal);
```

Comment:
> Well.. Passage will fail before this comes to Camus

what passage has to do with camus config? if passage should fail, why it doesn't now? :)